src/Components/Navbar.js

```diff
@@ -17,16 +19,28 @@ const Navbar = () => {
 
   return (
     <nav className="navbar flex-row">
-      <h1>Bookstore</h1>
-      <ul id="menu" className="navmenu flex-row">
-        {menuLinks.map((item) => (
-          <li key={item.id}>
-            <Link to={item.path}>
-              {item.name}
-            </Link>
-          </li>
-        ))}
-      </ul>
+      <div className="flex-row">
+        <h1 id="logo" className="blueText">Bookstore CMS</h1>
+        <ul id="menu" className="navmenu flex-row">
+          {menuLinks.map((item) => (
+            <li key={item.id}>
+              <Link to={item.path}>
+                {item.name}
+              </Link>
+            </li>
+          ))}
+        </ul>
+      </div>
+      <div className="user">
```

Comment:
- [ ] Hi @AbdollahRaafat, let's once more make sure that the user icon  is align with the main container as seen on the screenshot below 

| Yours |   Zeplin |
| :----:   | :----:      |
|  ![Screenshot from 2022-08-20 21-35-20](https://user-images.githubusercontent.com/63087491/185765359-76218b19-adcd-4648-9417-49b4a77e53de.png) |  ![zeplin](https://user-images.githubusercontent.com/63087491/185765380-9f4c4c45-1c8d-42b8-aeb8-c6fb179f22ff.png) |